lib/cfn-model/model/references.rb

```diff
@@ -29,41 +29,43 @@ def self.resolve_value(cfn_model, value)
     end
   end
 
+  def self.resolve_resource_id(reference, attr='Arn')
```

Comment:
what is the basis for make Arn the default here?

Reply:
There really wasn't one.  :). I modified it to default to nil, which will be passed down to subsequent methods and will return nil if GetAtt attribute is searched.